src/health.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"><defs><path id="a" d="M7.5 3.5l5 10.5L16 8h4V6h-5l-2.5 4.5L7.5 0 4 6H0v2h5z"/></defs><use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 5)"/></svg>
```

Comment:
- Please remove unnecessary `<defs>` `</defs>`
- Remove the `id="a"`
- Remove `<use fill="#000" fill-rule="nonzero" xlink:href="#a" transform="translate(2 5)"/>`